source/evidencevariable/structuredefinition-EvidenceVariable.xml

```diff
@@ -568,5 +568,207 @@
+			<short value="The lowest permissible value of the variable"/>
+			<definition value="The lowest permissible value of the variable, used with variables that have a number-based datatype (with or without units)."/>
+			<min value="0"/>
+			<max value="1"/>
+			<type>
+				<code value="Quantity"/>
+			</type>
+		</element>
+		<element id="EvidenceVariable.constraint.maximumQuantity">
+			<path value="EvidenceVariable.constraint.maximumQuantity"/>
+			<short value="The highest permissible value of the variable"/>
+			<definition value="The highest permissible value of the variable, used with variables that have a number-based datatype (with or without units)."/>
+			<min value="0"/>
+			<max value="1"/>
+			<type>
+				<code value="Quantity"/>
+			</type>
+		</element>
+		<element id="EvidenceVariable.constraint.earliestDateTime">
+			<path value="EvidenceVariable.constraint.earliestDateTime"/>
+			<short value="The earliest permissible value of the variable"/>
+			<definition value="The earliest permissible value of the variable, used with variables that have a date/time-based datatype."/>
+			<min value="0"/>
+			<max value="1"/>
+			<type>
+				<code value="dateTime"/>
+			</type>
+		</element>
+		<element id="EvidenceVariable.constraint.latestDateTime">
+			<path value="EvidenceVariable.constraint.latestDateTime"/>
+			<short value="The latest permissible value of the variable"/>
+			<definition value="The latest permissible value of the variable, used with variables that have a date/time-based datatype."/>
+			<min value="0"/>
+			<max value="1"/>
+			<type>
+				<code value="dateTime"/>
+			</type>
+		</element>
+		<element id="EvidenceVariable.constraint.minimumStringLength">
+			<path value="EvidenceVariable.constraint.minimumStringLength"/>
+			<short value="The lowest number of characters allowed for a value of the variable"/>
+			<definition value="The lowest number of characters allowed for a value of the variable, used with variables that have a string-based datatype."/>
+			<min value="0"/>
+			<max value="1"/>
+			<type>
+				<code value="integer"/>
+			</type>
+		</element>
+		<element id="EvidenceVariable.constraint.maximumStringLength">
+			<path value="EvidenceVariable.constraint.maximumStringLength"/>
+			<short value="The highest number of characters allowed for a value of the variable"/>
+			<definition value="The highest number of characters allowed for a value of the variable, used with variables that have a string-based datatype."/>
+			<min value="0"/>
+			<max value="1"/>
+			<type>
+				<code value="positiveInt"/>
+			</type>
+		</element>
+		<element id="EvidenceVariable.constraint.code">
+			<path value="EvidenceVariable.constraint.code"/>
+			<short value="Rule for acceptable data values"/>
+			<definition value="A rule, such as a format or other expectation, for the data values."/>
+			<min value="0"/>
+			<max value="1"/>
+			<type>
+				<code value="CodeableConcept"/>
+			</type>
+		</element>
```

Comment:
The ticket indicates code should have an Example binding

Reply:
Added the binding.